apps/arweave/src/ar_mining_server.erl

```diff
@@ -598,13 +604,20 @@ prepare_solution(poa2, Candidate,
 		partition_upper_bound = PartitionUpperBound } = Candidate,
 	case read_poa(RecallByte2, Chunk2, MiningAddress) of
 		error ->
+			Modules = ar_storage_module:get_all(RecallByte2 + 1),
```

Comment:
right now the `read_poa` wrapper throws out some error information [here](https://github.com/ArweaveTeam/arweave/pull/563/files#diff-ea8e957c63acb7497b0ed4f9201b40f18e5d5b4fa371e67b4cff05850e4df854R796-R810)

What do you think about either passing that error information up to here so we can log it, or printing it out in the wrapper directly?